src/modules/task/index.js

```diff
@@ -0,0 +1,10 @@
+const mongoose = require('mongoose');
```

Comment:
Firstly, the modules are not in the src/modules folder, but in src/model. Second, you don't need a "task" folder. Name the file "task"